### DISCUSSION TITLE
Taskflow API not working | airflow version -- 2.5.1

### DISCUSSION THREAD

**Discussion by JainyJoy** [apache/airflow#29777] [Q&A] [answered]
Hi,
I am running airflow on docker containers which uses image - **apache/airflow:slim-2.5.1**
I'm facing an issue where I'm unable to run any of the taskflow API dags, even the tutorial task available in the docker is also not working. They all fail without any error logs. Please help me to debug this.
![image](https://user-images.githubusercontent.com/52598380/221485522-b7199199-cb66-4eab-9f9f-8947264f24c5.png)
![image](https://user-images.githubusercontent.com/52598380/221485635-6f0d64cd-2ffd-4795-bcd0-280f66cf141d.png)
![image](https://user-images.githubusercontent.com/52598380/221485752-f30138a6-88d9-450e-86a2-7dc551134b41.png)
NB : I haven't made any changes in the tutorial code.

**Comment by potiuk** [comment, marked as answer]
Airflow slim does not have a lot of optional packages installed. It's really a starting point for you to extend  the image with packages you need. It is not supposed to be run without any modifications. You likely need to look at other log files generated.

Try it with 'regular' image. If it works there - this is certainly the case. You must find out which packages you need (the logs should tell you) and install them in an image that you build FROM slim image.

That's also good exercise to learn where the logs are and how to check them. You will need it anyway to effectively monitor and diagnose your installation

**Reply by JainyJoy** [reply]
Yes it did work with the image **apache/airflow:2.5.1**. 
Initially, I had installed extra packages like psycopg2 and celery , which were missing for the slim image. But not sure how I did not see logs for other missing packages. 
Thanks for the quick solution.
